data/CMakeLists.txt

```diff
@@ -186,6 +186,12 @@ if(USE_XMLLINT)
   add_dependencies(darktablerc_file validate_darktableconfig_xml)
 endif(USE_XMLLINT)
 
+if (WIN32 AND EXISTS ${CMAKE_CURRENT_BINARY_DIR}/darktablerc)
+  #make sure line endings are UNIX like even on Windows
+  configure_file(${CMAKE_CURRENT_BINARY_DIR}/darktablerc ${CMAKE_CURRENT_BINARY_DIR}/darktablerc.tmp NEWLINE_STYLE LF)
```

Comment:
I'd be surprized to learn that you can not pass that to normal `configure_file()` call.
I.e. this should be done when configuring `${CMAKE_CURRENT_BINARY_DIR}/darktablerc`